docs/documentation/stories/handling-environment-variables.md

```diff
@@ -0,0 +1,102 @@
+# Handling environment variables
+
+## Configuring available environments
+
+`.angular-cli.json` contains an **environments** section.  By default, this looks like:
+
+``` json
+"environments": {
+    "dev": "environments/environment.ts",
+    "prod": "environments/environment.prod.ts"
+}
+```
+
+You can add additional environments as required.  To add a **test** environment, your configuration would look like:
```

Comment:
I think a "staging" environment might be more common and a better choice for an example.  "Test" might cause confusion with the "ng test" command.